scipy/stats/tests/test_marray.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import numpy as np
+from scipy import stats
+
+from scipy._lib._array_api import xp_assert_close
+from scipy.stats._stats_py import _xp_mean
+skip_backend = pytest.mark.skip_xp_backends
+
+
+def get_arrays(n_arrays, *, dtype='float64', xp=np, shape=(7, 8), seed=84912165484321):
+    marray = pytest.importorskip('marray')
+    xpm = marray._get_namespace(xp)
+
+    entropy = np.random.SeedSequence(seed).entropy
+    rng = np.random.default_rng(entropy)
```

Comment:
Oops. Borrowed from `marray`.
```suggestion
    rng = np.random.default_rng(seed)
```